timemachine/fe/atom_mapping.py

```diff
@@ -271,6 +271,19 @@ def _compute_bond_cores(mol_a, mol_b, marcs):
     return bond_core
 
 
+def _compute_marcs(mol_a, mol_b, bond_core):
+    a_edges = get_edges(mol_a)
+    b_edges = get_edges(mol_b)
+    marcs = np.zeros((len(a_edges), len(b_edges)), dtype=np.byte)
+    for a, b in bond_core.items():
+        e_a = a_edges.index(a)
```

Comment:
is the ordering here guaranteed?

(i.e. bond [src,dst] and [dst,src] are properly canonicalized to be present in a_edges)

Reply:
Will take another approach that removed this code.